pyproject.toml

```diff
@@ -27,8 +27,9 @@ classifiers = [
 ]
 
 dependencies = [
-  "haystack-ai",
+  "haystack-ai==2.20.0",
```

Comment:
Let's make sure not to pin the haystack version. This causes issues when installing things like haystack-core-integrations. Basically pinning haystack-ai here causes other packages that instal haystack to always install this version. @anakin87 knows more about this.